reference/managed-delivery/resource-status/index.md

```diff
@@ -0,0 +1,160 @@
+
+If automatic actions successfully resolve the drift, the status will change to [Happy](/reference/managed-delivery/resource-status/#happy). If automatic actions don't help to resolve the drift, the status will change to [Unhappy](/reference/managed-delivery/resource-status/#unhappy).
+
+While Spinnaker is managing a resource, manual changes made via the UI or the API will be automatically reversed ("stomped") in favor of the declarative configuration.
+
+### Paused
+{%
+  include
+  figure
+  image_path="./resource-status-paused.png"
+  caption="Example of a resource with the Paused status"
+%}
+
+When a resource has a Paused status, it means Spinnaker is configured to continuously manage the resource but you've chosen to temporarily pause management. While management is paused Spinnaker won't check for drift from the declarative configuration, and it won't take any automatic actions. You can go to the Config view in your app to pause and resume management across the entire application.
+
+When you resume management, Spinnaker will check for drift from the declarative configuration. If a drift is detected, the status will change to [Diff](/reference/managed-delivery/resource-status/#diff). If no drift is detected, the status will change to [Happy](/reference/managed-delivery/resource-status/#unhappy).
+
+### Unknown
+{%
+  include
+  figure
+  image_path="./resource-status-unknown.png"
+  caption="Example of a resource with the Unknown status"
+%}
+
+When a resource has an Unknown status, it means Spinnaker was unable to determine the current status for some reason. Because the status is unknown, it's tough to tell by the status alone whether something is *wrong*, or just unexpected.
+
+After encountering this status, Spinnaker will continue trying to check the resource for its status and may eventually succeed. If it does, the resource will transition to whatever status is applicable at that time.
+
+If you see this status, it's probably best to investigate the current state of the resource manually and double check that nothing is broken. If you decide manual intervention is needed to remedy any issues you might discover, you can temporarily pause management on the Config view for your app.
+
+### Error
+{%
+  include
+  figure
+  image_path="./resource-status-error.png"
+  caption="Example of a resource with the Error status"
+%}
+
+When a resource has an Error status, it means something went wrong while trying to check the resource for drift from its declarative configuration. Because something went wrong, Spinnaker can't take automatic actions and drift from the declarative configuration won't be resolved. Some examples of problems that might lead to an error status:
```

Comment:
```suggestion
When a resource has an Error status, it means something went wrong while trying to check the resource for drift from its declarative configuration. Because something went wrong, Spinnaker can't take automatic actions and drift from the declarative configuration won't be resolved (until something changes). Some examples of problems that might lead to an error status:
```
The current phrasing sounds like we hard fail and never check the resource again (even though you say something else on 143). Would like to make this a bit more clear here.